### PR TITLE
Replace references to SilbinaryWolf with Symbiote (including folder structure).

### PR DESCRIPTION
**Pull request by JoshuaCarter**
The primary justification for doing this is that it makes comparing/maintaining the separate SS3/SS4 branches much easier. Also it's nicer and more consistent.

The main question here is whether it should be 1.3 or something else.

### CODE REVIEW

**Comment by nglasl** [comment]
I'm trying to figure out the best approach here, because these changes break backwards compatibility, so they technically need a major version bump if we're to follow semantic versioning. However, as you mentioned to me earlier, 2.X and 3.X is already taken, and it's weird having 4.X for SS3 when SS4 is 3.X. I'm thinking we don't really have much of a choice though, as it's either:

- keep things how they are prior to this change
- introduce this change and bump to 4.X
- upgrade the project you're working on to support SS4, so you only need to make the changes for SS4

I'm not getting much from reading around on the internet either, as this seems like a fairly unique scenario.

**Comment by nglasl** [comment]
As discussed, there's no elegant solution here. 4.X feels confusing and incorrect, but follows semantic versioning. 1.3 feels better, but doesn't follow semantic versioning and could cause issues for anyone trying to do an SS3 minor version bump. Considering that:

- This is mostly used internally at the moment, and isn't widely used among the open source community (according to reporting).
- SS3 is nearing end of life, so most projects should be aiming towards SS4.

I would opt to go with **1.3** as you recommended here. That way somebody intentionally needs to update the minor version before they encounter the namespace change, at the very least.

**Comment by JoshuaCarter** [comment]
@nglasl Many thanks for your thoughts (and external discussion). I agree and will go ahead with 1.3

**Comment by nglasl** [comment]
@JoshuaCarter you may have already done this, but could you please do a quick comparison against https://github.com/symbiote/silverstripe-components/commit/e92be4ef23f2efff49c8bbce5c16cc67bfbfac14 just to make sure you got everything that the master branch has? Aside from that, looks good to go from my perspective :)

**Review by nglasl — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -21,21 +21,24 @@
...
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "1.2.x-dev"
+            "dev-master": "1.3.x-dev"
         },
         "installer-name": "components"
     },
+    "replace": {
+        "silbinarywolf/silverstripe-components": ">=0.1.0 <=1.2.1"
```
Comment (JoshuaCarter) [inline composer.json:40]: @nglasl can I just run this by you, not entirely sure if it's sensible.

**Comment by JoshuaCarter** [comment]
I've made a couple minor changes to LICENSE that I missed from master.

**Review by JoshuaCarter — COMMENTED** [review]
Approved IRL

**Pull request merged**